target/scripts/startup/setup-stack.sh

```diff
@@ -453,6 +453,12 @@ function _setup_postfix_sizelimits
   postconf -e "virtual_mailbox_limit = ${POSTFIX_MAILBOX_SIZE_LIMIT}"
 }
 
+function _setup_clamav_sizelimit
+{
+  _notify 'inf' "Configuring clamav message scan size limit"
```

Comment:
As this would only how in debug log, would it be helpful to add some additional context with what value it is using?

```suggestion
  _notify 'inf' "Configuring clamav message scan size limit to ${CLAMAV_MESSAGE_SIZE_LIMIT}"
```